src/Main.java

```diff
@@ -1,6 +1,128 @@
+        // Задача 2
+
+        float floatNumber = 27.12f;
+        long longNumber = 987678965549L;
+        double doubleNumber = 2.786;
+        short shortNumber1 = 569;
+        short shortNumber2 = -159;
+        int intNumber = 27897;
+        byte byteNumber = 67;
+
+        // Задача 3
+
+        byte numberOfStudentsLP = 23;
+        byte numberOfStudentsAS = 27;
+        byte numberOfStudentsEA = 30;
+        short totalNumberOfSheets = 480;
+        int numberOfSheetsPerStudent = (totalNumberOfSheets / (numberOfStudentsLP
+                + numberOfStudentsAS + numberOfStudentsEA));
+        System.out.println("На каждого ученика рассчитано " + numberOfSheetsPerStudent
+                + " листов бумаги.");
+
+        // Задача 4
+
+        byte amountBottles = 16;
+        byte time = 2;
+        int machinePerformanceInMin = amountBottles / time;
+        byte period1 = 20;
+        short period2 = 24 * 60;
+        short period3 = 24 * 60 * 3;
+        int period4 = 24 * 60 * 30;
+        System.out.println("За " + period1 + " минут машина произвела "
+                + machinePerformanceInMin * period1 + " штук бутылок.");
+        System.out.println("За " + period2 + " минут машина произвела "
+                + machinePerformanceInMin * period2 + " штук бутылок.");
+        System.out.println("За " + period3 + " минут машина произвела "
+                + machinePerformanceInMin * period3 + " штук бутылок.");
+        System.out.println("За " + period4 + " минут машина произвела "
+                + machinePerformanceInMin * period4 + " штук бутылок.");
+
+        // Задача 5
+
+        byte totalAmountPaintCans = 120;
+        byte amountWhitePaintCansPerClass = 2;
+        byte amountBrownPaintCansPerClass = 4;
+        int amountClasses = (totalAmountPaintCans / (amountWhitePaintCansPerClass
+                + amountBrownPaintCansPerClass));
+        int amountWhitePaintCans = (amountClasses * amountWhitePaintCansPerClass);
+        int amountBrownPaintCans = (amountClasses * amountBrownPaintCansPerClass);
+        System.out.println("В школе, где " + amountClasses + " классов, нужно "
+                + amountWhitePaintCans + " банок белой краски и " + amountBrownPaintCans
+                + " банок коричневой краски.");
+
+        // Задача 6
+
+        byte amountBananas = 5;
+        byte weightOneBanana = 80;
+        short amountMilk = 200;
+        byte portionMilk = 100;
+        byte weightPortionMilk = 105;
+        byte amountIceCreams = 2;
+        byte weightOneIceCream = 100;
+        byte amountEggs = 4;
+        byte weightOneEgg = 70;
+
+        int weightBananas = (amountBananas * weightOneBanana);
+        int weightMilk = (amountMilk / portionMilk * weightPortionMilk);
+        int weightIceCreams = (amountIceCreams * weightOneIceCream);
+        int weightEggs = (amountEggs * weightOneEgg);
+        int weightBreakfastInGrams = (weightBananas + weightMilk + weightIceCreams
+                + weightEggs);
```

Comment:
Внешние скобки лишние. То есть можно писать вот так:
```suggestion
        int weightBananas = amountBananas * weightOneBanana;
        int weightMilk = amountMilk / portionMilk * weightPortionMilk;
        int weightIceCreams = amountIceCreams * weightOneIceCream;
        int weightEggs = amountEggs * weightOneEgg;
        int weightBreakfastInGrams = weightBananas + weightMilk + weightIceCreams
                + weightEggs;
```

Reply:
В остальных местах аналогично